src/main/java/roomescape/controller/ReservationController.java

```diff
@@ -10,32 +10,48 @@
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
-import roomescape.Reservation;
-import roomescape.Reservations;
+import roomescape.domain.Reservation;
+import roomescape.domain.ReservationTime;
+import roomescape.dto.ReservationResponse;
+import roomescape.dto.ReservationTimeResponse;
+import roomescape.repository.ReservationRepository;
+import roomescape.repository.ReservationTimeRepository;
 
 @RestController
 @RequestMapping("/reservations")
 public class ReservationController {
+    private final ReservationRepository reservationRepository;
+    private final ReservationTimeRepository reservationTimeRepository;
 
-    private final Reservations reservations = new Reservations();
-    private final AtomicLong index = new AtomicLong(1);
-
-    @GetMapping()
-    ResponseEntity<List<Reservation>> read() {
-        return ResponseEntity.ok(reservations.getReservations());
+    @Autowired
+    public ReservationController(ReservationRepository reservationRepository, ReservationTimeRepository reservationTimeRepository) {
+        this.reservationRepository = reservationRepository;
+        this.reservationTimeRepository = reservationTimeRepository;
     }
 
     @PostMapping
-    ResponseEntity<Reservation> create(@RequestBody Reservation reservation) {
-        Reservation newReservation = Reservation.toEntity(reservation, index.getAndIncrement());
-        reservations.add(newReservation);
+    ResponseEntity<ReservationResponse> create(@RequestBody Reservation reservation) {
+        Reservation createdReservation = reservationRepository.createReservation(reservation);
+        ReservationTime reservationTime = reservationTimeRepository.readReservationTime(createdReservation.getTimeId());
+
+        ReservationTimeResponse reservationTimeResponse = ReservationTimeResponse.of(reservationTime);
+        return ResponseEntity.ok(ReservationResponse.of(createdReservation, reservationTimeResponse));
+    }
+
+    @GetMapping()
```

Comment:
```suggestion
    @GetMapping
```
값이 없는 경우 `()`를 생략할 수 있어요~

Reply:
수정하겠습니다. 꼼꼼하게 짚어주셔서 감사합니다 :)